packages/data-stores/src/queries/use-site-goals.ts

```diff
@@ -0,0 +1,17 @@
+import { useQuery } from 'react-query';
+import wpcomRequest from 'wpcom-proxy-request';
+
+export function useSiteGoals( siteId: string | number | undefined ) {
```

Comment:
If I remember correctly this hook should be available in the Editing Toolkit, I tried to access this inside a React plugin there and looks like it's not accessible. There is a `useSiteIntent` hook available on Editing Toolkit, maybe this could be a good reference for this case and it's used on `SellerCelebrationModal` component.
File Path: `apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/lib/site-intent/use-site-intent.js`

Reply:
If it isn't the idea for this PR we can leave as it is and work on a new one to make it available on Editing Toolkit, totally fine.

Reply:
You are right, the issue was about adding the hook to editing toolkit. Checking. Thanks for pointing out!